hack/samples/index.json

```diff
@@ -119,6 +119,11 @@
     "description": "TerraGrow is an AI-powered app that provides crop recommendations and yield improvement strategies by analyzing environmental factors like soil nutrients, temperature, humidity, and pH. It helps farmers optimize planting decisions for higher yields.",
     "gitUrl": "https://github.com/daytonaio/sample-crops-ml"
   },
+  {
+    "name": "Sampel React.js - Tutor Connect",
```

Comment:
Sample instead of Sampel
Also, please add a space between : and " in the two properties below

Reply:
Done.